src/Common/Common.Api/Common.Api.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Microsoft.PowerBI.Api" Version="2.10.1925601" />
+    <PackageReference Include="Microsoft.PowerBI.Api" Version="1.0.0" />
```

Comment:
Will be bumped up...

Reply:
Can you send me via Teams the internal build so I can upload?